### PR TITLE
Created Directory 'Introduction' and moved related files to it

### PR DESCRIPTION
**Pull request by Bjane658**
So that all parent-directories of the subject are uniform, I changed 'Overview' to 'Introduction'.

### CODE REVIEW

**Comment by judepereira** [comment]
@Bjane658 You also need to setup redirects since this is a URL change. Can you set them up up please?

**Comment by Bjane658** [comment]
@judepereira ah I totally missed on that. Instead of changes in the file "_redirects" I would change the URL's in ".vuepress/config.js". Can you confirm?

**Comment by Bjane658** [comment]
... and of course in ".vuepress/components/Dashboard.vue" for the "Get Started" Button

**Comment by judepereira** [comment]
@Bjane658 You need to update the URLs in config.js, and _redirects. _redirects will list the new URLs, and _redirects will link the old link to the new one. Also, for some reason the deploy preview is failing, I'll check why.

**Comment by Bjane658** [comment]
@judepereira ok thanks, the "_redirect" is only left to change. Will change them.

File: _redirects [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # Redirect domain aliases to primary domain
 https://microservices.guide/* https://microservice.guide/:splat 301!
+
+# Redirect old introduction structure to new one
+https://microservices.guide/overview/ https://microservices.guide/introduction/overview/
```
Comment (judepereira) [inline _redirects]: @Bjane658 Do relative links work here? Can we try them out?
```suggestion
/overview/ /introduction/overview/
```

**Comment by judepereira** [comment]
That way, we can test out the redirect in the preview deployment as well :)

**Comment by Bjane658** [comment]
@judepereira the redirects seem to work fine, I update the other ones.

**Comment by judepereira** [comment]
@Bjane658 Looks like the deploy preview failed :( https://app.netlify.com/sites/microservice-guide/deploys/5c51d2ad6f66c200080f79df

**Comment by judepereira** [comment]
Sorry, ignore my last comment about the deploy preview.

**Comment by judepereira** [comment]
@Bjane658 looks like something is wrong. The redirect isn't working:
https://deploy-preview-53--microservice-guide.netlify.com/overview/

**Comment by Bjane658** [comment]
@judepereira maybe the full URL needs to be provided?
Why are we trying to provide the redirect, since the old URL is not present in the code anymore?

**Comment by judepereira** [comment]
@Bjane658 the old URL might be linked from other sources, and most importantly, might be cached by search engines. Hence, we must redirect the old URL to the new one.

**Comment by Bjane658** [comment]
@judepereira all right, good point. I change the URL's again in the _redirect. Then we will see.

**Comment by Bjane658** [comment]
@judepereira the redirect is still not working.

**Comment by judepereira** [comment]
@Bjane658 well the full URL would never work since it's trying to match microservice.guide, and the deploy preview is deploy-preview-53--microservice-guide.netlify.com.

According to Netlify, relative URL redirects should work:
https://www.netlify.com/docs/redirects/

**Comment by judepereira** [comment]
Can you revert your last commit, and then we can get in touch with Netlify to check what's wrong here.

**Comment by judepereira** [comment]
I've contacted Netlify support, and I'm awaiting their response.

**Comment by judepereira** [comment]
The redirects work now. I had to update the build command from `npm run build` to `npm run build && cp _redirects .vuepress/dist/`.

**Pull request merged**